community/io/src/test/java/org/neo4j/test/rule/PageCacheAndDependenciesRule.java

```diff
@@ -45,17 +46,19 @@
 
     public PageCacheAndDependenciesRule()
     {
-        this( () -> new EphemeralFileSystemRule() );
+        this( () -> new EphemeralFileSystemRule(), fs -> TestDirectory.testDirectory( fs ) );
```

Comment:
Can be simplified to:
`this( EphemeralFileSystemRule::new, TestDirectory::testDirectory );`

Reply:
`TestDirectory` will create it's own default file system if not provided with one and we want to use the same one as the page cache does

Reply:
The proposed simplification does not change behavior